features/step_definitions/edit_book_steps.rb

```diff
@@ -0,0 +1,15 @@
+When('I am on the Edit Book Page') do
+  book = Book.find_by(title: 'React')
+  visit(edit_book_path(book))
+end
+
+Given('the following book exists') do |table|
+  # table is a Cucumber::MultilineArgument::DataTable
+  table.hashes.each do |book|
+    Book.create!(book)
+  end
+end
+
+Then('I change {string} from {string} to {string}') do |field, _value1, value2|
```

Comment:
use the words `original_value` and `replacement_value` as descriptive variable names